tests/main/cgroup-tracking-failure/task.yaml

```diff
@@ -56,7 +56,7 @@ execute: |
     find /run/user/0 >run.txt
     case "$SPREAD_SYSTEM" in
         ubuntu-1[68].04-*)
-            systemctl --user stop dbus.{socket,service}
+            systemctl --user stop dbus.{socket,service} || true
```

Comment:
(nitpick) a small comment, maybe to the card even would be nice here

Reply:
I can do this in a follow-up, but it's not clear to me what you mean by "the card even would be nice here"